### PR TITLE
Mark online devrooms more clearly

### PR DESCRIPTION
**Pull request by johanvdw**
To test you should first update pentabarf.yml as well, ie 

```
nanoc update -y
nanoc
```

It will replace "d.energy" by "Online energy" (etc)  on all schedule pages and hide nav/capacity/building links.

It will not change any URL's, ~~but will change the room names also in the exported xml - not sure we want that. On one hand it is clearer, on the other hand, some app developers may already have done something to mark online devrooms based on their names.~~. Also the generated xml and ical are not changed.

### CODE REVIEW

**Comment by johanvdw** [comment]
image

image
image

**Comment by johanvdw** [comment]
I've attached a diff: no xml is changed, only html where the name changes.

[output-diff.txt](https://github.com/FOSDEM/website/files/10494320/output-diff.txt)

File: content/schedule/pentabarf-tz.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@ xml.schedule do
     xml.day(:index=>index + 1,:date=>day[:conference_day]) do
       rooms.each do | room |
 
-        xml.room({:name => room[:title]}) do
+        xml.room({:name => room[:conference_room]}) do
```
Comment (johanvdw) [inline content/schedule/pentabarf-tz.xml:33]: This actually fixes a bug - in all other part of the xml the room is linked to by its conference_room, and not its name

Comment (johanvdw) [reply content/schedule/pentabarf-tz.xml:33]: it should be fixed even if we don't change the titles now

**Comment by markvdb** [comment]
AFAIK, for video, we only use the xml.

**Comment by johanvdw** [comment]
> AFAIK, for video, we only use the xml.

I guess most apps do, because the yaml is only available on the machine that actually builds the website.

**Pull request merged**